api/v1alpha08/sonataflowplatform_types.go

```diff
@@ -111,6 +136,8 @@ type SonataFlowPlatformStatus struct {
 	// Triggers list of triggers created for the SonataFlowPlatform
 	//+operator-sdk:csv:customresourcedefinitions:type=status,displayName="triggers"
 	Triggers []SonataFlowPlatformTriggerRef `json:"triggers,omitempty"`
+	//+operator-sdk:csv:customresourcedefinitions:type=status,displayName="dbMigrationStatus"
+	SonataFlowPlatformDBMigrationStatus *SonataFlowPlatformDBMigrationStatus `json:"sonataFlowPlatformDBMigrationStatus,omitempty"`
```

Comment:
Please change this to `Phase` instead. It's not a Kubernetes Status API implementation.
```suggestion
	SonataFlowPlatformDBMigrationPhase *SonataFlowPlatformDBMigrationPhase `json:"sonataFlowPlatformDBMigrationPhase,omitempty"`
```